### ISSUE TITLE
Script data is reset when script is reloaded

### ISSUE THREAD

**Issue by ItsNathanG** [PlaceholderAPI-Expansions/Javascript-Expansion#3]
**Referencing Build #14**

The data that is passed to the placeholder scripts seems to be an entirely new data every time. 

Reproducible example:
```Javascript
var dataLoc = "%player_name%.viewed";

function viewCount() {
	var views = Data.exists(dataLoc) ? Data.get(dataLoc) : 0;

	views++;

	Data.set(dataLoc, views);
	Placeholder.saveData();

	return views;
}

viewCount();
```

This stores how many times a player has viewed the placeholder. The script itself works fine until PAPI is reloaded or the server is restarted.

**Comment by ItsNathanG** [comment]
Looks like when the script is loaded, [getData() is called](https://github.com/PlaceholderAPI-Expansions/Javascript-Expansion/blob/master/src/main/java/com/extendedclip/papi/expansion/javascript/JavascriptPlaceholder.java#L75), but the data is always null so a new script data is created.

Note that the data is loaded [here](https://github.com/PlaceholderAPI-Expansions/Javascript-Expansion/blob/master/src/main/java/com/extendedclip/papi/expansion/javascript/JavascriptPlaceholdersConfig.java#L180-L182), but by the time this is reached a new data was already created above.

**Comment by extendedclip** [comment]
Fixed 385dcc9af3fa6bed95e8030ac6afc810093129fa
